tests/e2e/pom.xml

```diff
@@ -0,0 +1,139 @@
+        <groupId>org.eclipse.che</groupId>
+        <version>7.29.0-SNAPSHOT</version>
+        <relativePath>../../pom.xml</relativePath>
+    </parent>
+    <groupId>org.eclipse.che.e2e</groupId>
+    <artifactId>che-e2e</artifactId>
+    <name>Che e2e tests</name>
+    <build>
+        <finalName>e2e-war</finalName>
+        <plugins>
+            <plugin>
+                <groupId>com.mycila</groupId>
+                <artifactId>license-maven-plugin</artifactId>
+                <configuration>
+                    <useDefaultExcludes>false</useDefaultExcludes>
+                    <excludes>
+                        <exclude>**/.idea/**</exclude>
+                        <exclude>**/*.styl</exclude>
+                        <exclude>**/*.html</exclude>
+                        <exclude>**/*.ico</exclude>
+                        <exclude>**/*.ttf</exclude>
+                        <exclude>**/*.eot</exclude>
+                        <exclude>**/*.css</exclude>
+                        <exclude>**/*.woff</exclude>
+                        <exclude>src/app/colors/che-output-colors.constant.ts</exclude>
+                        <exclude>src/app/proxy/proxy-settings.constant.ts</exclude>
+                    </excludes>
+                </configuration>
+            </plugin>
+            <plugin>
+                <artifactId>maven-antrun-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>remove-old-resources</id>
+                        <phase>clean</phase>
+                        <goals>
+                            <goal>run</goal>
+                        </goals>
+                        <configuration>
+                            <target>
+                                <exec dir="${basedir}" executable="docker" failonerror="true">
+                                    <arg value="run" />
+                                    <arg value="-v" />
+                                    <arg value="${basedir}/:/root/:Z" />
+                                    <arg value="node:10" />
+                                    <arg value="su" />
+                                    <arg value="-c" />
+                                    <arg value="cd /root; rm -rf dist node_modules" />
+                                </exec>
+                            </target>
+                        </configuration>
+                    </execution>
+                    <execution>
+                        <id>install-build-lint</id>
+                        <phase>compile</phase>
+                        <goals>
+                            <goal>run</goal>
+                        </goals>
+                        <configuration>
+                            <target>
+                                <!-- build tests and test them for lint -->
+                                <exec dir="${basedir}" executable="docker" failonerror="true">
+                                    <arg value="run" />
```

Comment:
specify `---user` with current user id if it's for file permissions issue

Reply:
Thank you for the advice, could you please help me with how can I obtain/use the user id in this code? I'm not fluent with poms, always fighting with it. Thank you